run-automation-test.md

```diff
@@ -0,0 +1,169 @@
+- **API key** 
+  + Click *user icon* -> **"Settings"**
+
+  ![get api key](./assets/api_key.png )
+### 2.3 Get desired capabilities
+The desired capabilities need to be added to the automation test script to allow the test to be executed on Kobiton device.
+  + Go to https://portal.kobiton.com, click **Devices** in the top navigation bar.
+ 
+    ![click devices](./assets/devices.png )
+ 
+  + Hover over any device you want to test with and click on the Automation settings button (the gear symbol).
+ 
+    ![click auto](./assets/click_auto.png )
+  
+  + On the left-hand side, you can select your preferred language, App Type, etc.
+  In this example, we use **NodeJS** as the default language of the script. Therefore, choose **NodeJS** in `Language` section and **Hybrid/Native from Apps** in `App type` section. On the right-hand side, Kobiton automatically translates your desired capabilities into code based on the options you selected. Copy the code to prepare for the next step.
+
+  ![auto](./assets/automation.png ) 
+## 3. Configure automation test script
+### 3.1. Fetching the app download URL
+To get the app download URL, we have provided a script written in NodeJS. After executing that script, you will get the latest app version, which will be used in the next step.
+
+Downloads the scipt from [this link](./getAppUrl.js) and copy to your automation test folder.
+### 3.2 Configure automation test script
+Kobiton has already provided sample scripts for automation testing, visit [here](https://github.com/kobiton/samples) for reference. 
+> Note: In this guideline, we will use the Node.js sample (samples/javascript folder) as an example. In sample/javascript folder, we will use `android-app-test` script. You can choose `android-app-test.js` if you want to run tests on Android or `ios-app-test.js` if on iOS.
+
+Use Kobiton’s desired capabilities settings to configure your test scripts. Our settings include all the required configurations you need to automate your test using the Appium Framework and to run your automation test on real devices in the Kobiton cloud.
+
+Open automation test script file in your repository or create a new one.
+
+Replace `desiredCaps` in the script with the ones collected in the previous step. Then,replace value of `app` element in `desiredCaps` with the `APP_URL` environment variable (The value of `app` element is the download URL of your app. If your automation test script is in other languages, please use the proper method to access this environment variable).
+
+Example in Node.js:
+
+```javascript
+var desiredCaps = {
+  sessionName:        'Automation test session',
+  sessionDescription: '', 
+  deviceOrientation:  'portrait',  
+  captureScreenshots: true, 
+  app:                process.env.APP_URL, 
+  deviceGroup:        'KOBITON', 
+  deviceName:         '<DEVICE_NAME>',
+  platformVersion:    '<DEVICE_VERSION>',
+  platformName:       '<DEVICE_PLATFORM_NAME>' 
+}
+```
+### 3.3 Configure TeamCity project
```

Comment:
I don't see the part where we mention the user's deploying process. Please at least mention or show a picture of 2 build step to highlight that the testing step occurs after deploying step.